datacats/cli/create.py

```diff
@@ -97,14 +96,14 @@ def init(opts):
     """Initialize a purged environment or copied environment directory
 
 Usage:
-  datacats init [-in] [--syslog] [-s NAME] [--address=IP] [ENVIRONMENT_DIR [PORT]]
-  datacats init [-ni] [--syslog] [--address=IP] [ENVIRONMENT_DIR [PORT]]
+  datacats init [-in] [--syslog] [-s NAME] [--address=IP]
+                [ENVIRONMENT_DIR [PORT]]
```

Comment:
I think `init` is a good fit for what it does. It's like `def __init__` in python where we have a class and now we're creating an instance from it. It also sounds like less work than `create` (fewer letters :smile:)